packages/suite/src/components/firmware/ReconnectDevicePrompt.tsx

```diff
@@ -105,7 +105,11 @@ const Heading = styled(H1)`
     font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
 `;
 
-const getTextForMode = (requestedMode: 'bootloader' | 'normal', deviceVersion: number) => {
+const getTextForMode = (
+    requestedMode: 'bootloader' | 'normal',
+    deviceModel: '1' | 'T',
+    deviceFwVersion: string,
+) => {
```

Comment:
In the previous commit we refactored several components to take the whole device, instead of individual model and version params. Any reason not to do the same thing here?

Reply:
I was thinking about it before but I decided not to do it. 

However, now it is changed (5d826e40b7a39fa598d27ad785d1ae084b5844ce)

Reply:
ok. .) Resolving. Merge this when you're ready. .)